packages/shared/styles/styleUtils/modifiers/width.ts

```diff
@@ -0,0 +1,7 @@
+import { css } from "react-emotion";
+
+export const width = width => {
```

Comment:
Not sure if `width` really needs to be a modifier. We don't have a predetermined collection of values to pick from like we do with modifiers like `padding`.